src/main.js

```diff
@@ -1,29 +1,40 @@
-var server = require('http').createServer();
-var io = require('socket.io')(server);
-import {Monopoly} from "./monopoly.js";
-import {PlayerList} from "./player.js";
+import http from 'http';
+import socketio from 'socket.io';
+import Monopoly from "./monopoly";
 
+const server = http.createServer();
+const io = socketio(server);
+const MonopolyGame = new Monopoly();
```

Comment:
In general, instances of classes should start lowercase and classes should start uppercase.  That way it's easier to visually inspect code and know exactly what's a variable and what's a class.